.circleci/config.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     steps:
       - checkout
       - run: mix --version
+      - run: node -v
+      - run: npm -v
```

Comment:
While adding a test for supporting ESM I saw that 
Elixir 15 -> nodejs 18.x
Elixir 16 -> nodejs 20.x

18 is about to lose LTS on 30th of April and 20 next year